Errors initializing cache database on GH actions

```
streaming api => ./mod.test.ts:804:6
error: AssertionError: Values are not equal.


    [Diff] Actual / Expected


-   Could not initialize cache database 'C:\Users\runneradmin\AppData\Local\deno\dep_analysis_cache_v1', deleting and retrying... (disk I/O error\n
-   \n
-   Caused by:\n
-       Error code 2570: I/O error within xDelete of a VFS object)\n
-   Failed to open cache file 'C:\Users\runneradmin\AppData\Local\deno\dep_analysis_cache_v1', opening in-memory cache.\n
    1\n
    2


  throw new AssertionError(message);
        ^
    at assertEquals (https://deno.land/std@0.182.0/testing/asserts.ts:189:9)
    at file:///D:/a/dax/dax/mod.test.ts:820:5
```

Version 1.33.1

https://github.com/dsherret/dax/actions/runs/4839977662/jobs/8625333975

--- a/cli/cache/cache_db.rs
+++ b/cli/cache/cache_db.rs
@@ -261,7 +261,9 @@ impl CacheDB {
     };
 
     // Failed, try deleting it
-    log::warn!(
+    let is_tty = atty::is(atty::Stream::Stderr);
+    log::log!(
+      if is_tty { log::Level::Warn } else { log::Level::Trace },
       "Could not initialize cache database '{}', deleting and retrying... ({err:?})",
       path.to_string_lossy()
     );
@@ -275,7 +277,12 @@ impl CacheDB {
 
     match self.config.on_failure {
       CacheFailure::InMemory => {
-        log::error!(
+        log::log!(
+          if is_tty {
+            log::Level::Error
+          } else {
+            log::Level::Trace
+          },
           "Failed to open cache file '{}', opening in-memory cache.",
           path.to_string_lossy()
         );
@@ -284,7 +291,12 @@ impl CacheDB {
         ))
       }
       CacheFailure::Blackhole => {
-        log::error!(
+        log::log!(
+          if is_tty {
+            log::Level::Error
+          } else {
+            log::Level::Trace
+          },
           "Failed to open cache file '{}', performance may be degraded.",
           path.to_string_lossy()
         );

